.stickler.yml

```diff
@@ -0,0 +1,10 @@
+# add the linters you want stickler to use for this project
+linters:
+  stylelint:
+    # indicate where is the config file for stylelint
+    config: 'stylelint.config.js'
```

Comment:
- [ ] `stylelintconfig.js` is a config file for css linting. You need to use ruby linter (rubocop) instead. Please make sure that you have the right configuration from [here](https://github.com/microverseinc/linters-config/tree/master/ruby)

Also make sure that you have added the `.rubocop.yml` file and is specified as the entry for this file.